tests/config_test.go

```diff
@@ -498,16 +498,16 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				res, err := expecter.ExpectBatch([]expect.Batcher{
 					// mount iso Secret image
 					&expect.BSnd{S: "sudo su -\n"},
-					&expect.BExp{R: "#"},
+					&expect.BExp{R: "\\#"},
```

Comment:
some of the prompts have `\\# ` some doesnt have the space, maybe best to keep one method ?
or at least in other PR where there will be a define for the prompt as we spoke please

Reply:
Yes, we defiantly should do a cleanup PR for the prompts issue.

Reply:
as Edy also said in this PR, lets please at least line up the changes that this PR did ?

Reply:
This is not changing the existing, so I do not think it is a must here. It can come in a following change.

Reply:
`\\` were added, it doesnt count ?